docker/run-opendj.sh

```diff
@@ -1,16 +1,31 @@
 #!/usr/bin/env bash
 
-# check if mysql is running
-CONTAINER=opendj
-RUNNING=$(docker inspect -f {{.State.Running}} $CONTAINER || echo "false")
-
-# mysql set-up
-if ! $RUNNING; then
-    echo "attempting to remove old opendj container..."
-    docker rm -f $CONTAINER || echo "docker rm failed. nothing to rm."
-
-    echo "starting up  container..."
-    docker run --name $CONTAINER -e ROOTPASS="testtesttest" -e BASE_DN=dc=dsde-dev,dc=broadinstitute,dc=org -v ${PWD}/docker/opendjsetup.sh:/opt/opendj/bootstrap/setup.sh -v ${PWD}/docker/example-v1.json:/opt/example-v1.json -d -p 3389:389 broadinstitute/openam:opendj
+start()  {
+    echo "starting up container $1..."
+    docker run --name $1 -e ROOTPASS="testtesttest" -e BASE_DN=dc=dsde-dev,dc=broadinstitute,dc=org -v ${PWD}/docker/opendjsetup.sh:/opt/opendj/bootstrap/setup.sh -v ${PWD}/docker/example-v1.json:/opt/example-v1.json -d -p 389 broadinstitute/openam:opendj
```

Comment:
so how do you actually know what port to connect to?

Reply:
if you don't specify one it picks a random free one on the host side and maps it to 389 on the container side